packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter_tools/src/application_package.dart';
 import 'package:flutter_tools/src/base/dds.dart';
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/features.dart';
+import 'package:flutter_tools/src/resident_devtools_handler.dart';
```

Comment:
The diffs in this file are due to adding a noop handler and removing all expectations. Aside from the fact that there is too much going in this test, the cases themselves did not prevent any bugs so I've chosen simplification.

The resident_devtools_handler_test.dart file contains tests for the invariants it needs to maintain